providers/terraform-provider-csbsqlserver/go.mod

```diff
@@ -0,0 +1,74 @@
+module github.com/cloudfoundry/csb-brokerpak-azure/terraform-provider-csbsqlserver
```

Comment:
Just curios.
Was converting this folder into a full-fledged Go module really needed or just an opportunistic improvement?